easybuild/easyconfigs/b/BRAKER/BRAKER-2.1.2-intel-2019a.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'Tarball'
+
+name = 'BRAKER'
+version = '2.1.2'
+
+homepage = 'https://github.com/Gaius-Augustus/BRAKER'
+description = """BRAKER is a pipeline for fully automated prediction of protein coding genes with GeneMark-ES/ET
+ and AUGUSTUS in novel eukaryotic genomes."""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
+
+source_urls = ['https://github.com/Gaius-Augustus/BRAKER/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['9f178c5fe64ae358dcba9936802d24e330312e698a3f7930d1f91e58974129d3']
+
+dependencies = [
+    ('Perl', '5.28.1'),
+    ('AUGUSTUS', '3.3.2'),
+    ('GeneMark-ET', '4.38'),
+    ('BamTools', '2.5.1'),
+    ('SAMtools', '1.9'),
+    ('GenomeThreader', '1.7.1', '', True),
```

Comment:
The version suffix `-Linux_x86_64-64bit` is missing here for `GenomeThreader`

Reply:
fixed in d826c0c9bf